examples/oic.r.acl-example2.json

```diff
@@ -29,61 +29,6 @@
           }
         ]
       }
-    ],
```

Comment:
Re: subject vs subjectuuid

I did try to update all the examples. But of course, any of the "acl" examples should not have been touched - since that would imply we broke backward compatibility of the JSON.

The ACE(1) "aces" entries all have subjectuuid. (and apparently always have had subjectuuid):

```
  "definitions": {
    "oic.sec.ace": {
      "type": "object",
      "properties": {
        "resources": {
          "type": "array",
          "description": "References the application's resources to which a security policy applies",
          "items": { "$ref": "../../core/schemas/oic.oic-link-schema.json#/definitions/oic.oic-link" }
        },
        "permission": {
          "type": "integer",
          "description": "Bitmask encoding of CRUDN permission",
          "$ref": "oic.sec.crudntype.json#/definitions/oic.sec.crudntype/properties/bitmask"
        },
        "validity": {
          "type": "array",
          "description": "validity is an array of time-pattern objects",
          "items": {
            "$ref": "oic.sec.time-pattern.json#/definitions/time-pattern"
          }
        },
        "subjectuuid": {
          "description": "The id of the device to which the ace applies to or \"*\" for anonymous access",
          "anyOf": [
            {
              "type": "string", "pattern": "^\\*$"
            },
            {
              "$ref": "../../core/schemas/oic.types-schema.json#/definitions/uuid"
            }
          ]
        }
      },
      "required": [ "resources", "permission", "subjectuuid" ]
    }
  }
```